app/Http/Controllers/VotesController.php

```diff
@@ -204,4 +205,36 @@ public function removeVote(Request $request, $id)
         $this->destroyVote($id);
         return AdministrationController::votes();
     }
-}
+
+    public function checkFakeEmails()
+    {
+        $api_key = env('API_KEY');
```

Comment:
Which api key? Mailing service? Hosting? Kittens gif api service?

Reply:
Anything you want and need